docs/guides/unbound.md

```diff
@@ -52,82 +52,154 @@ The first thing you need to do is to install the recursive DNS resolver:
+    access-control: 0.0.0.0/0 refuse
+    access-control: ::0/0 refuse
+    # Allow lo0
+    access-control: 127.0.0.0/8 allow
+    access-control: ::1 allow
+    # Allow lan
+    access-control: 192.168.0.0/16 allow
+    access-control: 172.16.0.0/12 allow
+    access-control: 10.0.0.0/8 allow
+    access-control: fd80:1fe9:fcee::/48 allow
+
+    # Prints one line per query to the log, making the server (significantly) slower
+    #log-queries: no
+
+    # Prevent banner disclosure
+    hide-identity: yes
+    hide-version: yes
+    # Refuse trustanchor.unbound queries
+    hide-trustanchor: yes
+
+    # The maximum dependency depth that Unbound will pursue in answering a query (reduced memory)
+    target-fetch-policy: "2 1 0 0 0 0"
+
+    harden-short-bufsize: yes
+    harden-large-queries: yes
+
+    # Use 0x20-encoded random bits in the query to foil spoof attempts
+    use-caps-for-id: yes
+
+    # DNSSEC bogus answers against DNS Rebinding
+    # RFC1918 private IP address space not allowed to be returned for public internet names
     private-address: 192.168.0.0/16
-    private-address: 169.254.0.0/16
     private-address: 172.16.0.0/12
     private-address: 10.0.0.0/8
+    private-address: 169.254.0.0/16
     private-address: fd00::/8
     private-address: fe80::/10
+    # Turning on 127.0.0.0/8 would hinder many spamblocklists as they use that
+    #private-address: 127.0.0.0/8
+    # Adding ::ffff:0:0/96 stops IPv4-mapped IPv6 addresses from bypassing the filter
+    private-address: ::ffff:0:0/96
+    # add 
+    private-address: 2001:470:b35c::/48
+
+    unwanted-reply-threshold: 10000000
+
+    do-not-query-localhost: no
+
+    # Affects buffer space
+    #prefetch: no
+
+    # UDP EDNS reassembly buffer advertised to peers. Default 4096.
+    # May need lowering on broken networks with fragmentation/MTU issues,
+    # particularly if validating DNSSEC.
+    #
+    #edns-buffer-size: 1480
+
+    # Use TCP for "forward-zone" requests. Useful if you are making
+    # DNS requests over an SSH port forwarding.
+    #
+    #tcp-upstream: yes
+
+    # DNS64 options, synthesizes AAAA records for hosts that don't have
+    # them. For use with NAT64 (PF "af-to").
+    #
+    #module-config: "dns64 validator iterator"
+    #dns64-prefix: 64:ff9b::/96 # well-known prefix (default)
+    #dns64-synthall: no
+
+    # Validation failure log level 2: query, reason, and server
+    val-log-level: 2
+
+    # Synthesise NXDOMAIN from nsec/nsec3 without hitting the authoritative
+    aggressive-nsec: yes
+
+    # Local copy of the full root zone, fallback to resolving from root servers
+    # "get off my lawn: if a lot of people were doing this it could considerably
+    # reduce the load on root nameservers and could increase resiliency in case of
+    # a dDOS attack on the root zone" --Florian Obser
+    # https://tools.ietf.org/html/draft-ietf-dnsop-7706bis-01
+    # https://github.com/wkumari/draft-kh-dnsop-7706bis/pull/8
+    auth-zone:
```

Comment:
I don't think we should hardcode then root zone IPs addresses. See my comment above.

Reply:
The auth zone is downloaded by Unbound, and that's the whole point of the RFC 7706.

> The primary goals of this design are to provide more reliable answers
>    for queries to the root zone during network attacks, and to prevent
>    queries and responses from being visible on the network.  This design
>    will probably have little effect on getting faster responses to stub
>    resolver for good queries on TLDs, because the TTL for most TLDs is
>    usually long-lived (on the order of a day or two) and is thus usually
>    already in the cache of the recursive resolver; the same is true for
>    the TTL for negative answers from the root servers.  (Although the
>    primary goal of the design is for serving the root zone, the method
>    can be used for any zone.)
> 
>    This document describes a method for the operator of a recursive
>    resolver to have a complete root zone locally, and to hide these
>    queries from outsiders.  The basic idea is to create an up-to-date
>    root zone server on the same host as the recursive server, and use
>    that server when the recursive resolver looks up root information.
>    The recursive resolver validates all responses from the root server
>    on the same host, just as it would all responses from a remote root
>    server.